shashi/jan28.js

```diff
@@ -0,0 +1,55 @@
+//Create a function that takes a string and returns a new string with all vowels removed.
+//Assumptions:
+//Only string input
+//No numbers
+
+var vowels = ["a", "e", "i", "o", "u"];
+
+function removeVowels(str = "") {
+  var newStr = "";
+  for (let i = 0; i < str.length; i++) {
+    for (let j = 0; j < vowels.length; j++) {
+      if (str[i].toLowerCase() === vowels[j]) {
+        break;
+      } else if (j === vowels.length - 1) {
+        newStr += str[i];
+      }
+    }
```

Comment:
Looks good, two things.... 
1. what other datatypes can store vowels?
2. For this implementation,  create a separate function for the inner for loop called isVowel(char).